apps/cnquery/cmd/login.go

```diff
@@ -63,13 +68,26 @@ You remain logged in until you explicitly log out using the 'logout' subcommand.
 		apiEndpointOverride, _ := cmd.Flags().GetString("api-endpoint")
 		err := register(token, annotations, timer, splay, apiEndpointOverride)
 		if err != nil {
+			if err == tokenValidationErr {
+				log.Error().Msg(err.Error())
+
+				// Prevents help message from being printed
+				return nil
+			}
 			defer func() {
 				s, err := checkStatus()
 				if err != nil {
 					log.Warn().Err(err).Msg("could not run status command")
 				}
 				s.RenderCliStatus()
 			}()
+
+			if err == tokenExpiredErr {
```

Comment:
Doesn't prevent check command from running to check possible "clock skew" as per https://github.com/mondoohq/cnspec/issues/1342

Reply:
I don't think we need to care about the complete status at this point. We already know that the token would not work.

When you have an expired token and no clock skew:
- You get this error message
- clock skew test would past
- But you get another error message here: https://github.com/mondoohq/cnquery/blob/main/apps/cnquery/cmd/status.go#L129

When you have an expired token and clock skew:
- You get this error
- You get the status error about the clock skew
- And you get the error from L129

As I understand the code and your comment:
We know the token has expired. Therefore, we can not use it to perform a full status check, including upstream. We might only check the clock skew and the latest version to provide the user with as much actionable information as possible. What do you think?

Reply:
Clock skew is detected by making the request (HealthProbe in this case) and comparing local clock to the server clock. Health probe should work with invalid token as well, and would highlight which API endpoint is attempted (issue we also had).